dcrjson/dcrwalletextresults.go

```diff
@@ -141,3 +143,12 @@ type WalletInfoResult struct {
 	VoteVersion      uint32  `json:"voteversion"`
 	Voting           bool    `json:"voting"`
 }
+
+// SweepAccountResult models the data returned from the sweepaccount
+// command.
+type SweepAccountResult struct {
+	UnsignedTransaction       []byte         `json:"unsignedTransaction"`
```

Comment:
binary data must be a hexadecimal string in the json-rpc api

Reply:
Additionally, the json struct tags must use all lowercase for consistency with the rest of the api.